config/atom.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
```

Comment:
Is there an issue about using cask to install Atom?  Would that make it possible to have a more comprehensive script?  Also, where's a check to see if the commands are available?